backend/clickhouse/query.go

```diff
@@ -771,11 +771,20 @@ func (client *Client) ReadMetrics(ctx context.Context, input ReadMetricsInput) (
 		return nil, err
 	}
 
+	// Base fields for sessions queries, needed if body filter is used - should clean this up, maybe make these top-level?
+	attributeFields := []string{"email", "device_id"}
```

Comment:
I noticed if I do a query `email EXISTS` or `email=spencer@highlight.io` that it returns no sessions. Not sure if that is intended?

Reply:
should be fixed now! was an issue with the sessions list, tested with email and other fields